test/presubmit-tests.sh.orig

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export DISABLE_MD_LINTING=1
+export DISABLE_MD_LINK_CHECK=1
+export DISABLE_YAML_LINTING=1
+
+source $(git rev-parse --show-toplevel)/vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
+
+<<<<<<< Updated upstream
+function check_go_lint() {
+    header "Testing if golint has been done"
+
+    # deadline of 5m, and show all the issues
+    golangci-lint -j 1 --color=never run
+
+=======
+export DISABLE_MD_LINTING=1
+export DISABLE_MD_LINK_CHECK=1
+export DISABLE_YAML_LINTING=1
+
+source $(git rev-parse --show-toplevel)/vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
+
+function check_go_lint() {
+    header "Testing if golint has been done"
+
+    # deadline of 5m, and show all the issues
+    golangci-lint -j 1 --color=never run
+
+>>>>>>> Stashed changes
```

Comment:
There is a conflict here :stuck_out_tongue:

Reply:
:stuck_out_tongue: 
fixed :+1: